### PR TITLE
TKWAR-20 Done

### PR DESCRIPTION
**Pull request by vogon101**
+Renderer Class
*OLD Renderer Deprecated
*Rendering re-written in Scala

### CODE REVIEW

File: src/com/vogon101/java/Renderer.scala [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import com.colofabrix.scala.tankwar.World
+import com.colofabrix.scala.tankwar.integration.TankEvaluator
+import org.lwjgl.LWJGLException
+import org.lwjgl.opengl.Display
+import org.lwjgl.opengl.DisplayMode
+import org.lwjgl.opengl.GL11
+import scala.collection.mutable.ListBuffer
+
+/**
+ * Created by Freddie on 17/05/2015.
```
Comment (ColOfAbRiX) [inline src/com/vogon101/java/Renderer.scala]: Describe the class

File: src/com/vogon101/java/Renderer.scala [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+      initGL()
+    }
+    catch  {
+      case lwjgle : LWJGLException => {println("Could not start Graphics"); System exit 1}
+    }
+
+    setCamera()
+  }
+
+  def update() {
```
Comment (ColOfAbRiX) [inline src/com/vogon101/java/Renderer.scala]: Public and protected methods must have a ScalaDoc or become private if there's no need to be public

File: src/com/vogon101/java/Renderer.scala [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+    drawTanks()
+    drawBullets()
+    Display.sync(25)
+    Display.update()
+
+    if (Display.isCloseRequested())
+      System.exit(0);
+  }
+
+  @throws(classOf[LWJGLException])
```
Comment (ColOfAbRiX) [inline src/com/vogon101/java/Renderer.scala]: Is this necessary in Scala?

Comment (vogon101) [inline src/com/vogon101/java/Renderer.scala]: How would you replace it?

File: src/com/vogon101/java/Renderer.scala (hunk 1 above)
Comment (vogon101) [inline src/com/vogon101/java/Renderer.scala]: I will add comments now

File: src/com/vogon101/java/Renderer.scala (hunk 3 above)
Comment (ColOfAbRiX) [inline src/com/vogon101/java/Renderer.scala]: I mean the throws annotation. In scala exceptions are not checked, you don't need to throw them. It's worth a quick investigation

Comment (vogon101) [inline src/com/vogon101/java/Renderer.scala]: I quite like leaving it in there so that one can see the exception thrown but it can be removed

**Pull request merged**